mvp.py

```diff
@@ -2,39 +2,51 @@
 from NearPoints import NearPoints
 from UberConnector import UberConnector
 from GeoConnector import GeoConnector
+import numpy as np
 
 UBER_SERVER_TOKEN = '7x_AZ2eWSn7nIekfylveYt3Hgb0juotM1JaPoawG'
 GEO_SERVER_TOKEN = 'TODO'
 START_POINT = (37.77, -122.41)
 END_POINT = (37.81, -122.41)
 NUM_NEIGHBOR_POINTS = 5
-RADIUS = .02
+RADIUS = .01 # in lat long units.
 
-def get_walking_dist():
-	# Use google maps to get walking dist between the two points.
-	return 0
+PRICE_WEIGHT = -20
+WALKING_TIME_WEIGHT = 3
+DRIVING_TIME_WEIGHT = 1
+
+def cost_function(min_price, max_price, walking_time, driving_time, wait_time = 0, ):
+	"""Cost is an aritrary unit that we desire to minimize.
+
+	Args:
+		min_price: minimum price of ride.
+		max_price: maximum price of ride.
+		walking_time: time to walk to uber pickup location.
+		driving_time: time spent in uber to destination.
+		wait_time: time spent between arrival at walking and uber pickup.
+
+	Returns:
+		cost of the trip.
+	"""
+	price = np.mean([max_price, min_price])
+	return PRICE_WEIGHT * price + WALKING_TIME_WEIGHT * walking_time + DRIVING_TIME_WEIGHT * driving_time
 
-def get_drive_time():
-	# Use google maps to get the driving time
-	return 0
 
 def main():
 	near_points = NearPoints(START_POINT[0], START_POINT[1])
 	uber_connector = UberConnector(UBER_SERVER_TOKEN)
 	geo_connector = GeoConnector(GEO_SERVER_TOKEN)
 	pts = near_points.get_nearby_points(NUM_NEIGHBOR_POINTS, RADIUS)
-	prices = set()
+	location_to_cost = {}
 	for pt in pts:
 		min_price, max_price = uber_connector.get_min_and_max(pt, END_POINT)
-		walking_time = geo_connector.get_walking_dist(START_POINT, pt)
-		drive_time = geo_connector.get_driving_time(pt, END_POINT)
-		#TODO(arthurbrant) use cost function
-		prices.add((min_price, max_price))
-
-	for price in prices:
-		print price
-
+		walking_time = geo_connector.get_walking_time(START_POINT, pt)
+		driving_time = geo_connector.get_driving_time(pt, END_POINT)
```

Comment:
a lot of this changes in my PR so let's just get that merged (since it doesn't already have merge conflicts) and then resolve the merges in this branch afterwards